packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -973,6 +998,27 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
     this._overflowItems = items;
   }
 
+  /** @private */
+  __getExtraWidth() {
+    // Check if max-width is set on the host
+    const maxWidth = parseInt(getComputedStyle(this).maxWidth);
+    if (!isNaN(maxWidth)) {
+      return maxWidth - this.clientWidth;
```

Comment:
This only works with pixel values for `max-width`. If you use for example `max-width: 50%` for the mscb, this will return a negative value and will not allow additional chips even though there would be space for it.

Reply:
Removed this logic.